ta4j-core/src/main/java/org/ta4j/core/num/Num.java

```diff
@@ -41,6 +41,21 @@
  */
 public interface Num extends Comparable<Num>, Serializable {
 
+    /**
+     * @return the Num of 0
+     */
+    Num zero();
```

Comment:
we could enhance javadoc to explain that this is used for caching common used constants

Reply:
The javadoc for interface should not contain implementation details.